onnxruntime/core/graph/graph.cc

```diff
@@ -1421,8 +1422,9 @@ Status Graph::InferAndVerifyTypeMatch(Node& node, const OpSchema& op) {
         // Logic error: This should not happen if we properly checked that every use has
         // a corresponding def, for which type-inference already produced a valid type
         Status status(ONNXRUNTIME, FAIL,
+                      "This is an invalid model. "
                       "Node (" + node_name + ") input arg (" +
```

Comment:
Not sure this is an invalid model given the comment says this failure is due to a logic error (i.e. our code not doing what it's meant to).

Reply:
Makes sense. Removed.